tests_and_analysis/test/euphonic_test/force_constant_utils.py

```diff
@@ -0,0 +1,306 @@
+from collections import namedtuple
+import json
+import numpy as np
+import numpy.testing as npt
+from euphonic import Crystal, ureg
+import os
+from tests_and_analysis.test.euphonic_test.test_crystal import (
+    check_crystal_attrs
+)
+from tests_and_analysis.test.utils import get_data_path
+import pytest
+from contextlib import contextmanager
+
+
+ConstructorArgs = namedtuple(
+    "ConstructorArgs",
+    [
+        "crystal", "force_constants", "sc_matrix",
+        "cell_origins", "born", "dielectric"
+    ]
+)
+
+ExpectedData = namedtuple(
+    "ExpectedData", ["fc_mat_cell0_i0_j0", "fc_mat_celln"]
+)
+
+
+class ExpectedForceConstants:
+
+    def __init__(self, dirpath: str, with_material: bool = True):
```

Comment:
Is there a reason we're loading from separate `.npy` files rather than just reading from a single .json file like in `test_crystal.py`? Tests would be easier to manage if they had a consistent structure...

Reply:
Absolutely true, refactored to use a single json file